### DISCUSSION TITLE
About parameters downloading.

### DISCUSSION THREAD

**Discussion by Kimi-Dai** [chaidiscovery/chai-lab#133] [Q&A] [answered]
I found that everytime I run the model it will download the parameters automaticlly.  However I find the parameter files actually has already been downloaded in site-package/download. Why is that and how can I solve it?
![image](https://github.com/user-attachments/assets/07dffb92-b5d3-44fe-94d4-38eaf6042f76)
![image](https://github.com/user-attachments/assets/a9dfdbf8-a37b-431c-98ba-d45b4da2f86f)

**Comment by arogozhnikov** [comment, marked as answer]
message is wrong, if file is there, it is not downloaded again.

Thanks for flagging.

**Reply by arogozhnikov** [reply]
fixed in #134
